packages/calling-stateful-client/src/CallAgentDeclarative.test.ts

```diff
@@ -80,8 +77,7 @@ class MockCallAgent implements CallAgent {
     return call;
   }
   join(groupLocator: GroupLocator, options?: JoinCallOptions): Call;
-  join(groupChatCallLocator: GroupChatCallLocator, options?: JoinCallOptions): Call;
-  join(meetingLocator: MeetingLocator, options?: JoinCallOptions): Call;
+  join(meetingLocator: TeamsMeetingLinkLocator, options?: JoinCallOptions): Call;
```

Comment:
I guess this one can be called teamsMeetingLocator? Should be fine cause this is just test

Reply:
And the one typed with any, will be just meetingLocator

Reply:
Keeping changes minimal.